doc/build-unix.md

```diff
@@ -5,39 +5,23 @@ file COPYING or http://www.opensource.org/licenses/mit-license.php.
 This product includes software developed by the OpenSSL Project for use in the [OpenSSL Toolkit](http://www.openssl.org/). This product includes
 cryptographic software written by Eric Young ([eay@cryptsoft.com](mailto:eay@cryptsoft.com)), and UPnP software written by Thomas Bernard.
 
-UNIX BUILD NOTES FOR PEERCOIN-QT
+UNIX BUILD NOTES FOR PEERCOIN
 ====================
 
-To Build in Terminal on Debian based Linux
+To build in Terminal on Debian based Linux
 ---------------------
 
-	sudo apt-get -y install build-essential libqt4-dev qt5-qmake cmake qttools5-dev libqt5webkit5-dev qttools5-dev-tools qt5-default python-sphinx texlive-latex-base inotify-tools libboost-dev libboost-system-dev libboost-filesystem-dev libboost-program-options-dev libboost-thread-dev openssl libssl-dev libdb++-dev libminiupnpc-dev git sqlite3 libsqlite3-dev g++ libpng-dev	
+	sudo apt-get install build-essential libtool autotools-dev autoconf automake pkg-config libssl-dev libdb-dev libdb++-dev libboost-all-dev libevent-dev bsdmainutils libqt5gui5 libqt5core5a libqt5dbus5 qttools5-dev qttools5-dev-tools libprotobuf-dev protobuf-compiler git
 	sudo apt-get -y update && sudo apt-get -y upgrade
 	mkdir -p ~/.peercoin
 	echo "rpcuser=username" >>~/.peercoin/peercoin.conf
 	echo "rpcpassword=`head -c 32 /dev/urandom | base64`" >>~/.peercoin/peercoin.conf
 	git clone https://github.com/peercoin/peercoin
 	cd peercoin
-	git checkout v0.6.1ppc
-	qmake -qt=qt5 && make
-
-
-UNIX BUILD NOTES FOR PEERCOIND
-====================
-
-To Build in Terminal on Debian based Linux
----------------------
-
-	sudo apt-get -y install build-essential libqt4-dev qt5-qmake cmake qttools5-dev libqt5webkit5-dev qttools5-dev-tools qt5-default python-sphinx texlive-latex-base inotify-tools libboost-dev libboost-system-dev libboost-filesystem-dev libboost-program-options-dev libboost-thread-dev openssl libssl-dev libdb++-dev libminiupnpc-dev git sqlite3 libsqlite3-dev g++ libpng-dev	
-	sudo apt-get -y update && sudo apt-get -y upgrade
-	mkdir -p ~/.peercoin
-	echo "rpcuser=username" >>~/.peercoin/peercoin.conf
-	echo "rpcpassword=`head -c 32 /dev/urandom | base64`" >>~/.peercoin/peercoin.conf
-	git clone https://github.com/peercoin/peercoin
-	cd peercoin
-	git checkout v0.6.1ppc
-	cd src
-	make -f makefile.unix -j4
+	git checkout v0.6.2ppc
+	./autogen.sh
+	./configure --with-incompatible-bdb
```

Comment:
This option is dangerous as it make old wallet.dat files incompatible.
Is there no way to install bdb4.8 on Debian?

Reply:
Tried this, but it did not work:

git clone https://github.com/peercoin/peercoin

cd peercoin

PEERCOIN_ROOT=$(pwd)

BDB_PREFIX="${PEERCOIN_ROOT}/db4"

mkdir -p $BDB_PREFIX

wget 'http://download.oracle.com/berkeley-db/db-4.8.30.NC.tar.gz'

echo '12edc0df75bf9abd7f82f821795bcee50f42cb2e5f76a6a281b85732798364ef  db-4.8.30.NC.tar.gz' | sha256sum -c

tar -xzvf db-4.8.30.NC.tar.gz

cd db-4.8.30.NC/build_unix/

../dist/configure --enable-cxx --disable-shared --with-pic --prefix=$BDB_PREFIX

make install

cd $PEERCOIN_ROOT

git checkout v0.6.2ppc

./autogen.sh

./configure LDFLAGS="-L${BDB_PREFIX}/lib/" CPPFLAGS="-I${BDB_PREFIX}/include/"

make

Reply:
I know it works with Bitcoin.

Reply:
It does not work for me either, something with the configure script.

On ubuntu if I use the dbd4.8 package compiles fine. Created a separate pull request.

Reply:
Can you update the build notes for v0.6.2 in doc/build-unix.md too when you find the solution?
When you do I will close down this pull.